lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -20,6 +20,12 @@ fabric_forwarding:
   get_value: '/^feature fabric forwarding$/'
   set_value: "feature fabric forwarding"
 
+itd:
+  _exclude: [N3k, N5k, N6k]
```

Comment:
We need to verify if this is supported on N8k.  I will send you an email with connection details to my N8k to verify the cli.